tests/integration/features/overlay_partitions/lifecycle/test_chisel_lifecycle.py

```diff
@@ -0,0 +1,20 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright 2022 Canonical Ltd.
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License version 3 as published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+# Make pytest run any non-overridden chisel tests here with partitions enabled.
+
+# pylint: disable=wildcard-import,function-redefined,unused-import,unused-wildcard-import
+from tests.integration.lifecycle.test_chisel_lifecycle import *  # noqa: F403
```

Comment:
clever!

Reply:
Thanks! I thought it was clever too, which was part of why I was uncomfortable doing it (clever code often being harder to reason about). I figured in this case the cleverness was about making it more robust though so it was okay as long as it had an explanation.